app/controllers/admins/users_controller.rb

```diff
@@ -19,7 +19,11 @@ def index
 
     def update
       if user.update(user_params)
-        redirect_to admins_user_path(user)
+        if params.dig(:user, :blocked).present?
```

Comment:
for now redirect to show page in any way

Reply:
done